README.md

```diff
@@ -44,11 +56,23 @@ Add props to the `ImageCarousel` component that controls the `showThumbs` and `s
 
 In your use of the `ImageCarousel` component for Task 1, make sure the thumbnails are not visible, but the indicators are visible.
 
-### 3. Style the `ImageCarousel` component
-Using the mockup included in the root level of the project, position and size the `ImageCarousel` component on the page, with considerations for how responsive it is on smaller screen sizes.
-
 ### 4. Add a loading indicator
 While waiting for the results from the image API to be returned and rendered, display a loading indicator. A spinner image can be found under the `public` directory. The spinner should be displayed at 100% of its size in the middle of the page.
 
+## Frontend Task
+### 1. Style the `ImageCarousel` component
+Using the mockup included in the root level of the project, position and size the `ImageCarousel` component on the page, with considerations for how responsive it is on smaller screen sizes.
+
+### 2. Add Navigation bar
+Make the navigation bar look similar to the mockup.jpg
+
+### 4. Response name and description 
+Make sure the names and description of the exact instead of {name} or {description}
```

Comment:
Is there a typo here? Also, I am not sure what the response and description are 🤔